postgres/src/test/java/org/jdbi/v3/postgres/TestLobStream.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.sql.SQLException;
+import java.util.function.Supplier;
+
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.postgresql.PGConnection;
+import org.postgresql.largeobject.LargeObjectManager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestLobStream {
+    private static final int BIGLY = 1024 * 1024 * 64;
```

Comment:
I'm missing an expression of what this is for and why it's the value that it is, e.g. `SAMPLE_SIZE_GREATLY_EXCEEDING_LIMIT` or a short comment?

Reply:
It's just a big amount of data that would cause problems to use inline.  Nothing more.

Reply:
Can we give it a serious name expressing that then instead of a cheeky one?

Reply:
Coming up with fun names is part of the otherwise not-fun of writing tests... I'll change this I guess but it really is just a "big" number for testing purposes.  Consider focusing your reviewing time on the meat of the issue at hand.  If you're not comfortable with the api, maybe checkout the upstream docs and learn a bit about it!  I didn't know anything about the Postgres LargeObject stuff before starting this PR.